src/pages/johyeoungjin/Login/JoLogin.js

```diff
@@ -1,7 +1,77 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+import './JoLogin.scss';
 
 function JoLogin() {
-  return <div>조형진 로그인페이지</div>;
+  const [idValue, setIdValue] = useState('');
+  const [pwValue, setPwValue] = useState('');
```

Comment:
```suggestion
  const [userInput, setUserInput] = useState({ id: '', password: '' });
```
- 하나의 객체로 관리해도 좋을 듯 합니다.